src/configure/configure.ts

```diff
@@ -280,15 +280,14 @@ class Orchestrator {
 
     private async analyzeNode(node: any): Promise<void> {
         if (!!node) {
-            if (await this.extractAzureResourceFromNode(node)) {
+            if (node.fsPath) {
```

Comment:
If we do above, then we can check the type here using `typeof`. This makes the code easier to understand